changelog_unreleased/javascript/pr-8926.md

```diff
@@ -1,4 +1,4 @@
-#### Support range ignore for javascript code
+#### Implement range ignore (javascript) ([#8926](https://github.com/prettier/prettier/pull/8926) by [@chaitan94](https://github.com/chaitan94))
```

Comment:
```suggestion
#### Implement range ignore ([#8926](https://github.com/prettier/prettier/pull/8926) by [@chaitan94](https://github.com/chaitan94))
```

Reply:
The changelog already categorized

Reply:
Makes sense, I removed the javascript part.